components/automate-ui/src/app/modules/team/team-add-users/team-add-users.component.ts

```diff
@@ -183,4 +187,11 @@ export class TeamAddUsersComponent implements OnInit, OnDestroy {
     this.openUserModal.emit(true);
   }
 
+  triggerFullPageMessage() {
+    // `fired` variable just minimizes browser's workload since that gets called continuously
```

Comment:
❓ What kind of optimization is that? Do we do that in many places?

Reply:
No, it is something new. See my soon-to-be-posted warning in #a2-team.

Reply:
the suspense!